common/src/main/java/com/netflix/conductor/common/run/WorkflowSummary.java

```diff
@@ -296,6 +301,14 @@ public void setPriority(int priority) {
         this.priority = priority;
     }
 
+    public String getCreatedBy() {
+        return createdBy;
+    }
+
+    public void setCreatedBy(String createdBy) {
+        this.createdBy = createdBy;
+    }
```

Comment:
We might need to put createdBy here also https://github.com/orkes-io/conductor/blob/main/common/src/main/java/com/netflix/conductor/common/run/WorkflowSummary.java#L300 ?